esmvalcore/config-developer.yml

```diff
@@ -278,3 +278,11 @@ EMAC:
   input_file: ''
   output_file: '[dataset]_[ensemble]_[short_name]_[start_year]-[end_year]'
   cmor_type: 'CMIP5'
+
+CORDEX:
+  input_dir:
+    default: '/'
+    drs1: '[project]/[product]/[Domain]/[Institution]/[GCMModelName]/[CMIP5ExperimentName]/[CMIP5EnsembleMember]/[RCMModelName]/[RCMVersionID]/[Frequency]/[VariableName]'
```

Comment:
We were working with these directories just for simplicity. If you want to increase the reproducibility and generalize the name, it is a good idea.

Reply:
If it is just your local standard path, the convention is to name the drs according to the super computer or computing center where it is used, eg BADC for the uk data center at ceda, DKRZ for the German Deutsches KlimaRechenZentrum, etc. Maybe a similar name would be appropriate for your use case?

Reply:
Sorry for the confusion... What I meant is that the default path was the one that we were using for simplicity, the other has been named as a combination of the characteristics of dataset and project.  They can be changed to the best user use, so I do not think it would be a problem. Even you can create another directory named drs2: ' ... ... ... ... ' where you will feel more comfortable, if you want. I think that the most important part in this pull request is how the input file name is given in order to avoid to choose several files at the same time.